addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -342,36 +425,74 @@ private void handleCcMediaStatus(final MediaStatus mediaStatus) {
                         PlayPauseType.PLAY);
                 break;
             default:
+                logger.debug("Unknown mediaStatus: {}", mediaStatus.playerState);
                 break;
         }
+
+        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_CURRENT_TIME),
+                new DecimalType(mediaStatus.currentTime));
+
+        updateMediaInfoStatus(mediaStatus.media);
     }
 
-    private void handleCcVolume(final Volume volume) {
-        PercentType value = new PercentType((int) (volume.level * 100));
-        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_VOLUME), value);
-        this.volume = value;
-        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_MUTE),
-                volume.muted ? OnOffType.ON : OnOffType.OFF);
+    private void updateMediaInfoStatus(final Media media) {
+        Double duration = 0.0;
```

Comment:
Same mention, UNDEF might be more appropriate than some default value. Or initialize with null then you catch it at line 448, but not on 450...